core/tauri-build/src/mobile.rs

```diff
@@ -48,13 +53,27 @@ dependencies {"
 
   app_build_gradle.push_str("\n}");
 
+  if let Some(version) = config.version.as_ref() {
+    app_gradle_properties.push_str(&format!("tauri.android.versionName={}\n", version));
+    if let Some(version_code) = config.bundle.android.version_code.as_ref() {
+      app_gradle_properties.push_str(&format!("tauri.android.versionCode={}\n", version_code));
+    } else if let Ok(version) = Version::parse(version) {
+      let version_code = version.major * 1000000 + version.minor * 1000 + version.patch;
+      app_gradle_properties.push_str(&format!("tauri.android.versionCode={}\n", version_code));
```

Comment:
we could emit `tauri.` prefix since it will be inside `tauri.properties` file
```suggestion
    app_gradle_properties.push_str(&format!("android.versionName={}\n", version));
    if let Some(version_code) = config.bundle.android.version_code.as_ref() {
      app_gradle_properties.push_str(&format!("android.versionCode={}\n", version_code));
    } else if let Ok(version) = Version::parse(version) {
      let version_code = version.major * 1000000 + version.minor * 1000 + version.patch;
      app_gradle_properties.push_str(&format!("android.versionCode={}\n", version_code));
```